src/status_im/contexts/wallet/common/scan_account/view.cljs

```diff
@@ -29,10 +30,12 @@
      {:title           (i18n/label :t/scan-qr)
       :subtitle        (i18n/label :t/scan-an-account-qr-code)
       :error-message   (i18n/label :t/oops-this-qr-does-not-contain-an-address)
-      :validate-fn     #(contains-supported-address? %)
+      :validate-fn     (fn [scanned-text]
+                         (contains-supported-address? scanned-text))
       :on-success-scan (fn [result]
                          (let [address (extract-address result)]
                            (when on-result (on-result address))
-                           (debounce/debounce-and-dispatch
-                            [:wallet/scan-address-success address]
-                            300)))}]))
+                           (when address
+                             (debounce/debounce-and-dispatch
+                              [:wallet/scan-address-success address]
+                              300))))}]))
```

Comment:
is the WC part handled here?

Reply:
No, this is some other QR code component. I mistook it for the global reader earlier.